files/scripts/sonic-dpu-mgmt-traffic.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+#Script to control the DPU management traffic forwarding through the SmartSwitch
+
+command_name=$0
+
+usage(){
```

Comment:
Where is the trigger to execute this script with enable/disable option? Is this part of pmon?

Reply:
There is no trigger for this script, this is just a utility script which would be helpful for enabling DPU traffic to reach the internet in cases where connectivity is required, Example Usage: 
SONiC to SONiC upgrade can be done easily if we enable DPU management traffic forwarding using this script, as the DPU can download the image directly, instead of first transferring the image to switch and then to DPU